test/movie_dsl_test.exs

```diff
@@ -36,6 +36,27 @@ defmodule PropCheck.Test.MoviesDSL do
       end
     end
   end
+
+  @tag will_fail: true
+  property "server has illegal states" do
```

Comment:
What's the point of this test? I think it just checks whether the post condition crashes.
If I remember correctly, I introduced the random crashing in `ask_for_popcorn` just as a way to inspect previous implementation.

Reply:
Yes, that's true, but this is exactly the reason. An easy toggle of failure, it is automatically excluded by tagging it with `@tag will_fail: true`, but we can check if error messages and the like does behave well.